companies/1and1-gb.json

```diff
@@ -19,5 +19,5 @@
     ],
     "request-language": "en",
     "suggested-transport-medium": "email",
-    "quality": "verified"
+    "quality": "tested"
```

Comment:
🚫 **[companylint]** <[tested-requirements](https://github.com/datenanfragen/data/issues/811)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record has `quality` of `tested` but doesn't specify `required-elements`.

Preferably set `required-elements`, otherwise demote to `verified`.
```suggestion
    "quality": "verified"
```